source/System.cpp

```diff
@@ -167,7 +167,7 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 				// that they are no longer here.
```

Comment:
```suggestion
				// that they are no longer here.
				// Use const_cast to convert the "const Planet *" to "Planet *".
				// Non-const access is available through the passed parameter "Set<Planet> &planets"
				// but, in the case of an as-yet undefined Planet, the object will not have a name with which
				// it can be found in that collection.
```